test/scripts/jenkins_security_solution_cypress_rac.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
I think this should have a license comment at the top?

Reply:
File no longer exists.